### ISSUE TITLE
TP Re-opening PR 

### ISSUE THREAD

**Issue by shirsho-12** [nus-cs2103-AY2223S2/forum#230]
I'd created a pull request for the TP but had named it incorrectly. The NUS SE-Bot stated this
" Your team org is named incorrectly. It should be AY2223S2-CS2103T-W09-2
If that is the case, you can reopen this PR after renaming the org with the correct name. "

How do I re-open this pull request? Or do I just create a new PR?

**Comment by damithc** [comment]
> How do I re-open this pull request? Or do I just create a new PR?

@shirsho-12 creating a new PR is fine too.

**Comment by jiasheng59** [comment]
I don't quite understand what issue you are facing, but I did some search on "re-open" a pull request.

I found that reopening a merged pull request is impossible (at least for 13 years ago). 
https://stackoverflow.com/questions/12674304/github-reopening-a-merged-pull-request

but there's a button named "revert" might be of help to create a new PR in order to revert the changes of that previous PR.
